frame/uniswap-v2/src/lib.rs

```diff
@@ -58,52 +58,15 @@ pub mod pallet {
 		PalletId,
 	};
 	use scale_info::TypeInfo;
-	use sp_core::crypto::KeyTypeId;
 	use sp_runtime::{
 		traits::{
 			AccountIdConversion, AtLeast32BitUnsigned, CheckedAdd, CheckedDiv, CheckedMul,
 			CheckedSub, IntegerSquareRoot, One, Zero,
 		},
-		FixedPointNumber, FixedPointOperand, FixedU128, KeyTypeId as CryptoKeyTypeId, Permill,
+		FixedPointNumber, FixedPointOperand, FixedU128, Permill,
 	};
 	use sp_std::{collections::btree_set::BTreeSet, fmt::Debug, iter::FromIterator};
 
-	pub const PALLET_ID: PalletId = PalletId(*b"Uni-swap");
-	pub const KEY_TYPE: KeyTypeId = KeyTypeId(*b"uswp");
-	pub const CRYPTO_KEY_TYPE: CryptoKeyTypeId = CryptoKeyTypeId(*b"uswp");
-
-	pub mod crypto {
-		use super::KEY_TYPE;
-		use sp_core::sr25519::Signature as Sr25519Signature;
-		use sp_runtime::{
-			app_crypto::{app_crypto, sr25519},
-			traits::Verify,
-			MultiSignature, MultiSigner,
-		};
-		app_crypto!(sr25519, KEY_TYPE);
-
-		pub struct TestAuthId;
-
-		// implementation for runtime
-		impl frame_system::offchain::AppCrypto<MultiSigner, MultiSignature> for TestAuthId {
-			type RuntimeAppPublic = Public;
-			type GenericSignature = sp_core::sr25519::Signature;
-			type GenericPublic = sp_core::sr25519::Public;
-		}
-
-		// implementation for mock runtime in test
-		impl
-			frame_system::offchain::AppCrypto<
-				<Sr25519Signature as Verify>::Signer,
-				Sr25519Signature,
-			> for TestAuthId
-		{
-			type RuntimeAppPublic = Public;
-			type GenericSignature = sp_core::sr25519::Signature;
-			type GenericPublic = sp_core::sr25519::Public;
-		}
-	}
-
```

Comment:
Is this module intentionally removed?

Reply:
yeah, it was there in case we needed it later, so better to reduce the attack surface and remove it